app/views/projects/projects/show.html.erb

```diff
@@ -1,21 +1,85 @@
-<h1><%= @project.title %></h1>
+<div class="row">
+<nav class="projects-nav">
```

Comment:
This `nav` should be indented as it's a child of the `div` above.

Reply:
Done